app/api/user.py

```diff
@@ -247,10 +251,36 @@ def create_user(
         if supplier_code:
             user_data['role'] = 'supplier'
         else:
+            business_info_values = ''
+            organisation_name = ''
+            state = ''
+            postcode = ''
+            check_international = abn.validate(abn)
+            if check_international is not False:
```

Comment:
This is a little confusing.  It looks like we're just validating the ABN.  Would this read better if we updated it to the following?
```suggestion
            validated_abn = abn.validate(abn)
            if validated_abn:
```

Reply:
resolved